dma.8

```diff
@@ -226,6 +226,8 @@ Uncomment if you want TLS/SSL secured transfer.
 Uncomment if you want to use STARTTLS.
 Only useful together with
 .Sq SECURETRANS .
+.It Ic FINGERPRINT Xo
+Optionally verify SHA256 fingerprint of smarthost's certificate.
```

Comment:
How about: "Pin the server certificate by specifying its SHA256 fingerprint.  Only makes sense if you use a smarthost."